pdns/ws-auth.cc

```diff
@@ -591,6 +580,208 @@ static void apiZoneCryptokeys(HttpRequest* req, HttpResponse* resp) {
     throw HttpNotFoundException();
   }
   resp->setBody(doc);
+
+}
+
+/*
+ * This method handles DELETE requests for URL /api/v1/servers/:server_id/zones/:zone_name/cryptokeys/:cryptokey_id .
+ * It deletes a key from :zone_name specified by :cryptokey_id.
+ * Server Answers:
+ * Case 1: zone_name not found
+ *      The server returns 400 Bad Request
```

Comment:
The Api spec does say [here](https://github.com/PowerDNS/pdns/blob/master/docs/markdown/httpapi/api_spec.md#common-error-causes) if an invalid `zone_id` is provided, `400 Bad Request ` should be returned, but we definitely need to return the same response for all methods.